src/components/basic_table/in_memory_table.js

```diff
@@ -132,18 +132,41 @@ const getInitialPagination = pagination => {
   };
 };
 
-const getInitialSorting = sorting => {
+function findColumnByProp(columns, prop, value) {
+  for (let i = 0; i < columns.length; i++) {
+    const column = columns[i];
+    if (column[prop] === value) return column;
```

Comment:
Absolutely just me being nit-picky, but I always use braces for `if` statements as I find it easier to visually grok.

```suggestion
    if (column[prop] === value) {
      return column;
    }
```